edge-hub/core/test/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter.Test/MqttBrokerConnectorTest.cs

```diff
@@ -322,6 +336,8 @@ public async Task OfflineSendGetSentAfterReconnect()
                 broker.DropActiveClient();
             }
 
+            await Task.Delay(TimeSpan.FromMilliseconds(200)); // give time to settle with dropped connection
```

Comment:
do we have a work item for a better fix?

Reply:
I am creating one